vic/drivers/shared_image/include/vic_mpi.h

```diff
@@ -32,6 +32,14 @@
 
 #define VIC_MPI_ROOT 0
 
+/******************************************************************************
+ *  * @brief   This structure stores netCDF file name and corresponding nc_id
+ *   *****************************************************************************/
+typedef struct {
+    char    nc_filename[MAXSTRING];
+    int     nc_id;
+} nameid_struct;
+
```

Comment:
I think that this needs to be moved to `vic_driver_shared_image.h` where all the other netcdf related structures are defined.